cloud/blockstore/libs/storage/partition_common/get_changed_blocks_companion.h

```diff
@@ -8,27 +8,15 @@ namespace NCloud::NBlockStore::NStorage {
 
 ////////////////////////////////////////////////////////////////////////////////
 
-// Helps to process the TEvGetChangedBlocksRequest message. It can either
-// respond to it with an error, or redirect it to another actor.
+// Helps to process the TEvGetChangedBlocksRequest message. It respond to
+// request with an error.
 class TGetChangedBlocksCompanion
```

Comment:
there is no state left in this class ... so maybe just replace this to function `DeclineGetChangedBlocks` and even move it to `part_nonrepl_common.cpp`